documents/scripts/utils.injector.js

```diff
@@ -0,0 +1,210 @@
+#!/usr/bin/env node
+import fs from 'node:fs';
+import path from 'node:path';
+import chalk from 'chalk';
+import { Build, generateDocList } from './paths.js';
+
+/*
+ * Return generated header with markdown syntax.
+ * @returns {string}
+ */
+const generateHeader = (text, level = 1) => {
+  let sharps = ''
+  for (let i = 0; i < level; i++) {
+    sharps += '#'
+  }
+  return `${sharps} ${text}\n\n`
+}
+const generateDescription = (description) => {
+  return `${description}\n\n`
+}
+const generateParameter = (parameters) => {
+  let result = `${generateHeader('Arguments', 4)}\n\n| Name | Type | Description |\n| --- | --- | --- |\n`;
+  if (parameters.length > 0) {
+    for (const param of parameters) {
+      result += `| ${param.name || ''} | ${param.type || ''} | ${param.description.replace("\n", " ").replaceAll("\r"," ") || ''} |\n`;
+    }
+  }
+  result += '\n\n';
```

Comment:
there is one `\n` already.
```suggestion
  result += '\n';
```